app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  Checkbox,
+  ScrollView,
+  Flex,
+  Button,
+  Box,
+  View,
+} from 'native-base'
+import StaRibbon from '../Components/StaRibbon'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([])
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) => {
+    return roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase())
+  })
+
+  return (
+    <>
+      <HStack mt={-10} mb={12} alignItems={'flex-end'}>
+        <Heading
+          size="md"
+          letterSpacing={3}
+          fontWeight={400}
+          mb={-3}
+          fontFamily="BebasNeue-Regular"
+          adjustsFontSizeToFit
+        >
+          My Experience
+        </Heading>
+        <Flex flex={1} alignItems={'flex-end'} mt={-10}>
+          <StaThemeLogo width={'100%'} />
+        </Flex>
+      </HStack>
+      <StaRibbon />
+      <View margin={'4'}>
+        <FreeSearchBar
```

Comment:
Would it be possible to make it so the user can come out of the free search input by tapping anywhere else on the screen?  The only way you can do that at the moment is to tap on one of the checkboxes.

I started searching and then wanted to see the full list of results, but some of the results were hidden underneath my keyboard.  If we could make it possible to 'blur' / lose focus from the text input by tapping elsewhere on the screen background that would make it possible to see all the checkboxes in the results list.

Reply:
Hi Joe, thanks for all the comments and very helpful. Will see to those changes  tomorrow. Just on the point about the search box I think I understand what you're asking but wanted to clarify. Are you meaning that when a user has typed something into the searchbox e.g. ''web developer" and then clicks _anywhere_ else on the screen (loses focus), all the checkboxes should '_reappear_' again with the searchbox being 'cleared'?

Reply:
Hi @DJMF2015 are you on an Android phone? I’m on an iPhone, I wonder if that could be the difference? When I tap outside the search box it doesn’t lose focus (the keyboard remains) — unless I tap on a checkbox. Here’s how it looks for me in this video

https://github.com/Scottish-Tech-Army/Volunteer-app/assets/25086222/c6f95da3-30df-4012-8c08-564b9ae96491

Reply:
(The part where nothing seems to be happening is me tapping on different parts of the screen)

Reply:
@joefhall thanks for providing that. I think it may well be an iphone issue as i am on android and it doesn't do it for me (when i click outside the text box the keyboard automatically disappears). I read there is something called `TouchableWithoutFeedback` which allows the keyboard to be dismissed when they user loses focus? Perhaps i can add this and see if it makes a difference. `TouchableWithoutFeedback` is a react-native specific module.

Reply:
@joefhall just added an attempt at a fix for the problem you are experiencing on ios. Let me know if it makes any difference as i only have android myself.

Reply:
Thanks @DJMF2015 unfortunately not -- it's still the same for me

Reply:
@DJMF2015 I've found a way that works on iPhone -- if you make the top-level element in MyExperienceContainer `<Pressable onPress={() => Keyboard.dismiss()}>` (with `</Pressable>` the very last line of what's returned).  (You'll need to import `Pressable` from NativeBase.)

Fingers crossed that works on Android too.